libs/components/phone-field/src/lib/modules/phone-field/phone-field.component.spec.ts

```diff
@@ -690,9 +690,12 @@ describe('Phone Field Component', () => {
     });
 
     describe('disabled state', () => {
-      it('should disable the input and dropdown when disable is set to true', () => {
+      it('should disable the input and dropdown when disable is set to true', async () => {
         component.isDisabled = true;
         fixture.detectChanges();
+        await fixture.whenStable();
```

Comment:
What's the change to `setDisabledState` that's causing the issue? Should we make a note to add to the migration guide?

Reply:
Seems like it must also be related to [this change](https://angular.io/guide/update-to-version-15#setdisabledstate-is-always-called-when-a-controlvalueaccessor-is-attached).